<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalSystemInfoDlg</class>
 <widget class="QDialog" name="LocalSystemInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblOsVersion">
       <property name="text">
        <string>OS Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="valOsVersion">
       <property name="text">
        <string>Windows NT 5.1 Build 2600 Service Pack 2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="grpCPU">
       <property name="title">
        <string>CPU</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lblProcNum">
          <property name="text">
           <string>Number of Processors</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="valProcNum">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblProcType">
          <property name="text">
           <string>Processor Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="valProcType">
          <property name="text">
           <string>586</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblPageSize">
          <property name="text">
           <string>PageSize</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="valPageSize">
          <property name="text">
           <string>4096</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblAllocationGranularity">
          <property name="text">
           <string>Allocation Granularity</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="valAllocationGranularity">
          <property name="text">
           <string>65563</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblAppAdrMin">
          <property name="text">
           <string>Minimum application Address</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="valAppAdrMin">
          <property name="text">
           <string>00010000</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblAppAdrMax">
          <property name="text">
           <string>Maximum application Address</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="valAppAdrMax">
          <property name="text">
           <string>7FFEFFFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="grpMemoryStatus">
       <property name="title">
        <string>Memory Status</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblTotalPhysical">
          <property name="text">
           <string>Total Physical</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="valTotalPhysical">
          <property name="text">
           <string>2038 MB</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblAvailPhysical">
          <property name="text">
           <string>Avail Physical</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="valAvailPhysical">
          <property name="text">
           <string>1534 MB</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblTotalPageFile">
          <property name="text">
           <string>Total Pages of Memory</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="valTotalPageFile">
          <property name="text">
           <string>3929 MB</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblAvailPageFile">
          <property name="text">
           <string>Avail Pages of Memory</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="valAvailPageFile">
          <property name="text">
           <string>3555 MB</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblTotalVirtual">
          <property name="text">
           <string>Total Virtual</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="valTotalVirtual">
          <property name="text">
           <string>2048 MB</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblAvailVirtual">
          <property name="text">
           <string>Avail Virtual</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="valAvailVirtual">
          <property name="text">
           <string>1984 MB</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QLabel" name="piePhysical">
          <property name="text">
           <string>Physical</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2" rowspan="2">
         <widget class="QLabel" name="piePageFile">
          <property name="text">
           <string>Page File</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" rowspan="2">
         <widget class="QLabel" name="pieVirtual">
          <property name="text">
           <string>Virtual</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QGroupBox" name="grpSocket">
       <property name="title">
        <string>Socket</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lblHostName">
          <property name="text">
           <string>Local host name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="valHostName">
          <property name="text">
           <string>ACER1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblHostAdr">
          <property name="text">
           <string>Local host address</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="valHostAdr">
          <property name="text">
           <string>192.168.1.100</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblSocketVer">
          <property name="text">
           <string>Socket version</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="valSocketVer">
          <property name="text">
           <string>2.2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblDescription">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="valDescription">
          <property name="text">
           <string>WinSock 2.2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblSysStatus">
          <property name="text">
           <string>System status</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="valSysStatus">
          <property name="text">
           <string>Running</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblIcmpImpl">
          <property name="text">
           <string>ICMP implementation</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="valIcmpImpl">
          <property name="text">
           <string>IPHLPAPI.DLL</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lblSocketMax">
          <property name="text">
           <string>Max Sockets</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="valSocketMax">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="lblUdpSize">
          <property name="text">
           <string>Max UDP packet size</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="valUdpSize">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="grpInternet">
       <property name="title">
        <string>Internet connection status</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lblConnection">
          <property name="text">
           <string>Connection detected</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="valConnection">
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblUsesLan">
          <property name="text">
           <string>System uses LAN</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="valUsesLan">
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblUsesModem">
          <property name="text">
           <string>System uses modem</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="valUsesModem">
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblUsesProxy">
          <property name="text">
           <string>System uses proxy</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="valUsesProxy">
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="grpIPv6">
       <property name="title">
        <string>IPv6</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblIPv6">
          <property name="text">
           <string>System is IPv6 ready</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="valIPv6">
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
